src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -162,5 +177,15 @@ private void OnViewClosed(object sender, EventArgs e)
                 doc.FileActionOccurred -= OnFileSaved;
             }
         }
+
+        private void AddErrorToList(IError error)
+        {
+            if (_errorList == null)
+            {
+                _errorList = new ErrorList(_project.Name, _manifestPath);
```

Comment:
I believe we were talking about _project.Name possibly being null and using a default project name in Output Window review? Would it apply here as well?

Reply:
Fixed